src/types/request/v1/futuresV1.ts

```diff
@@ -215,3 +215,5 @@ export interface HistoricPlanOrderTPSLRequest {
   isPre?: boolean;
   isPlan?: string;
 }
+
+export type FuturesCandleData = string[6];
```

Comment:
Would love a small tsdoc-style comment to say where in the array each candle prop is